docs/requirements-doc.txt

```diff
@@ -13,7 +13,8 @@ recommonmark
 sphinx<6.0.0
 sphinx-click
 # ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
-ray[default]>=2.1.0,!=2.5.0
+ray>=2.1.0,!=2.5.0
```

Comment:
```suggestion
ray-core>=2.1.0,!=2.5.0
```
?

Reply:
Analogue of conda package `ray-core` for pip - just `ray`.